### ISSUE TITLE
Esther Obonyo and USAID/PEER/ScholarSphere/Research Data

### ISSUE THREAD

**Issue by patriciahswe** [olendorf/data_crm#6]
Referred by Stephanie Sims of DMPTool (from email of 2/10/2016): "Esther contacted me about leveraging the DMPTool to support USAID researchers. She also has questions about repository selection that I thought you would be better qualified to address. Esther has explored institution-based options including ScholarSphere but is also seeking advice about where to direct foreign researchers to deposit research data funded by USAID."

Also wants to talk about PEER solicitation: http://sites.nationalacademies.org/cs/groups/pgasite/documents/webpage/pga_168694.pdf

Obonyo, Esther A.
Associate Professor of Engineering Design and Architectural Engineering
The Pennsylvania State University
213 Hammond Building
University Park, PA 16802
eao4@psu.edu

**Comment by olendorf** [comment]
Still working out a meeting time with Esther (If i follow the email thread right, is that correct Patricia?)

**Comment by patriciahswe** [comment]
@olendorf Yes, but if it turns out that Esther can't change the date/time, would you be willing to meet with her on your own? (I have to travel out of state on March 31 to spend a long weekend with family in CT - it's my mother's 81st birthday.)

**Comment by olendorf** [comment]
Absolutely I can. So we'll get it taken care of.

**Comment by olendorf** [comment]
I met with Esther on Friday 4/1/2015. Turns out her question wasn't so much about help with repositories so much as she is the lead in developing USAID open data policy. Our discussion was pretty wide ranging but I think the most interesting parts were

  * She wanted to discuss how to design a DMP requirement
  * She is interested in why open data is important, beyond that its required.
    * She was excited about benefits of open data - increased publications, faster science etc.
    * Also, she was excited about how it can benefit developing countries
  * We also talked a lot about ways to document data that doesn't involve complex metadata/ontologies.
 
Much of the discussion had to do with dealing with researchers in developing countries with the best of intentions, and limited resources, who are getting funding from USAID.
